WooCommerce/Classes/Extensions/UIColor+Tooltip.swift

```diff
@@ -0,0 +1,27 @@
+import UIKit
+
+extension UIColor {
+    static var invertedSystem5: UIColor {
+        return UIColor(light: UIColor.systemGray5.color(for: UITraitCollection(userInterfaceStyle: .dark)),
```

Comment:
nit: can omit `return` and `UIColor` like in `invertedLink` (same for other color properties in this file)
```suggestion
        UIColor(light: .systemGray5.color(for: UITraitCollection(userInterfaceStyle: .dark)),
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10005/commits/7cbe3330889bb345abcbb00fc228b26269b4af12